modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc

```diff
@@ -3,18 +3,21 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Custom policies need to be published to Exchange so users can obtain and apply them to APIs. API Manager lists custom policies published as assets of the Organization.
+You need to publish your custom policies to Exchange so your users can obtain and apply them to their APIs. +
+API Manager lists custom policies published as assets of the Organization.
```

Comment:
no need to capitalize Organization unless we are talking about the object specifically (Organization has seven fields...) Here we just mean an Anypoint Platform organization.